build-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2.3"
+
+services:
+  carla-build-machine:
+    image: carla-localbuildenv:latest
+    runtime: nvidia
+    privileged: true
+    security_opt:
+      - seccomp:unconfined
+    working_dir: '/home/carla/carla'
+    volumes:
+      - '.:/home/carla/carla'
+      - '/tmp/.X11-unix:/tmp/.X11-unix:rw'
```

Comment:
I think you also need to bind mount Vulkan driver 
volumes:
` - /etc/vulkan/icd.d:/etc/vulkan/icd.d`

otherwise, when launching the editor you will get this error
Can not find a compatible Vulkan driver
[problem link](https://github.com/carla-simulator/carla/issues/1977)

Reply:
> I think you also need to bind mount Vulkan driver volumes: ` - /etc/vulkan/icd.d:/etc/vulkan/icd.d`
> 
> otherwise, when launching the editor you will get this error Can not find a compatible Vulkan driver [problem link](https://github.com/carla-simulator/carla/issues/1977)


can you tell me what command i need to do instead of " - /etc/vulkan/icd.d:/etc/vulkan/icd.d" if I have a host windows machine running Nvidia rtx A2000?.

Reply:
Do you know what the hell Docker even does? It's a tool to virtualize **Linux** processes (!!!) ... Why on earth do you even remotely expect that it'll work on Windows? 😅😂

Come on, get outta here and install a Linux side boot. This poor high-end GPU wants to operate on a proper OS 😉😂😂😂

Reply:
The issue is that its a company laptop. It wont let me dual boot windows and ubuntu. So i have to figure out a way to run the carla container on windows 10. I have tried everything but unfortunately no success

Reply:
If you manage to access your GPU inside WSL2's Linux VM, I could imagine that there's a chance for it to work 😂

But I'd advise you to rent a GPU empowered workstation and access it via network 😉 (your salary for a few weeks is probably higher than those renting costs)

Reply:
Thanks for the hint. I will give this a shot